packages/terra-clinical-site/src/site.scss

```diff
@@ -25,7 +25,8 @@
       height: 100%;
     }
 
-    [class*="ItemView__accessory"] {
+    [class*="ItemView__accessory"],
+    [class*="ItemView__accessory"] > * {
```

Comment:
This seems a little out of place, what is this doing exactly?

This wildcard class selector is dependent on the css-loader options defined in the webpack config. If the option changes this css would stop working.